content/javascript/concepts/arrays/terms/toSpliced/toSpliced.md

```diff
@@ -0,0 +1,89 @@
+---
+ Title: '.toSpliced()'
+Description: 'Returns a new array with deleted, replaced, or inserted values at the given index.'
+Subjects:
+  - 'Web development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+## .toSpliced()
+
+**`.toSpliced()`** is a method that modifies multiple array elements in one go. It usually has a start point (the first element to be modified) and an end point (the last element to be modified). `.toSpliced()` alters array elements in the following ways: 
+
+- Delete element(s)
+- Replace element(s)
+- Insert element(s)
+
+> **Note:** `.toSpliced()` returns the copy of an array with the given modifications. It does not mutate the parent array.
+
+
+## Syntax
+```pseudo
+array.toSpliced(startIndex, count, elementN)
+```
+
+`.toSpliced` takes the following parameters:
+
+- `startIndex`: the location of the first element to be altered.
+
+- `count`: the number of elements that should be modified, beginning from `startIndex`
+
+- `elementN`: the updated value(s) for the array element(s). To insert multiple values, seperate each value with a comma. 
+
+
+## Examples
+#### Delete array elements with `.toSpliced`
+
+```js
+const colors = ['red', 'yellow', 'blue', 'orange', 'green', 'purple'];
+
+// Removing  red, yellow, and blue. Start and index 0 and delete 3 elements
```

Comment:
I agree with this suggestion. But it raises a new issue.

 If .toSpliced() extracts elements from one array and places them in another, then maybe it creates a new array - rather than modifies the original array. I'd say this logic applies to all the other copying methods as well. 

I'm committing the changes based on your current suggestions now. But I'll be happy to make more changes.